recipes/libsndio/all/conanfile.py

```diff
@@ -0,0 +1,153 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import chdir, copy, get, rmdir, replace_in_file, download
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.scm import Version
+from conan.tools.build import cross_building
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class LibsndioConan(ConanFile):
+    name = "libsndio"
+    license = "ISC"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sndio.org/"
+    topics = ("sndio", "sound", "audio", "midi")
+    description = "A small audio and MIDI framework that provides a lightweight audio & MIDI server \
+        and a user-space API to access either the server or the hardware directly in a uniform way."
+    package_type = "shared-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_alsa": [True, False]
+    }
+    default_options = {
+        "with_alsa": True,
+    }
+
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def validate(self):
+        if self.settings.os != "Linux":
```

Comment:
```suggestion
        if self.settings.os not in ["Linux", "FreeBSD"]:
```
I see it being available on FreeBSD (https://repology.org/project/sndio/versions) as well, so it's probably supported.